packages/labs/cli/src/lib/generate/generate.ts

```diff
@@ -30,7 +42,7 @@ export const run = async (
     await Promise.allSettled(
       importers.map(async (importer) => {
         const generator = await importer();
-        await generator.run(analysis, console);
+        await writeFileTree(outDir, await generator(analysis));
```

Comment:
What's the error handling case here? If we don't, I think we should have a custom error message instead of just printing a stack trace. Something like "Error writing files to foo/bar"?

Reply:
added TODO